website/redirects.js

```diff
@@ -0,0 +1,1234 @@
+  {
+    source: '/docs/commands/logs(.html)?',
+    destination: '/docs/commands/alloc/logs',
+    permanent: true,
+  },
+  {
+    source: '/docs/commands/init(.html)?',
+    destination: '/docs/commands/job/init',
+    permanent: true,
+  },
+  {
+    source: '/docs/commands/inspect(.html)?',
+    destination: '/docs/commands/job/inspect',
+    permanent: true,
+  },
+  {
+    source: '/docs/commands/run(.html)?',
+    destination: '/docs/commands/job/run',
+    permanent: true,
+  },
+  {
+    source: '/docs/commands/stop(.html)?',
+    destination: '/docs/commands/job/stop',
+    permanent: true,
+  },
+  {
+    source: '/docs/commands/plan(.html)?',
+    destination: '/docs/commands/job/plan',
+    permanent: true,
+  },
+  {
+    source: '/docs/commands/validate(.html)?',
+    destination: '/docs/commands/job/validate',
+    permanent: true,
+  },
+  {
+    source: '/docs/commands/client-config(.html)?',
+    destination: '/docs/commands/node/config',
+    permanent: true,
+  },
+  {
+    source: '/docs/commands/node-drain(.html)?',
+    destination: '/docs/commands/node/drain',
+    permanent: true,
+  },
+  {
+    source: '/docs/commands/node-status(.html)?',
+    destination: '/docs/commands/node/status',
+    permanent: true,
+  },
+  {
+    source: '/docs/commands/keygen(.html)?',
+    destination: '/docs/commands/operator/keygen',
+    permanent: true,
+  },
+  {
+    source: '/docs/commands/keyring(.html)?',
+    destination: '/docs/commands/operator/keyring',
+    permanent: true,
+  },
+  {
+    source: '/docs/commands/server-force-leave(.html)?',
+    destination: '/docs/commands/server/force-leave',
+    permanent: true,
+  },
+  {
+    source: '/docs/commands/server-join(.html)?',
+    destination: '/docs/commands/server/join',
+    permanent: true,
+  },
+  {
+    source: '/docs/commands/server-members(.html)?',
+    destination: '/docs/commands/server/members',
+    permanent: true,
+  },
+  {
+    source: '/docs/runtime/schedulers(.html)?',
+    destination: '/docs/schedulers',
+    permanent: true,
+  },
+  {
+    source: '/docs/internals/scheduling(.html)?',
+    destination: '/docs/internals/scheduling/scheduling',
+    permanent: true,
+  },
+
+  // Sometimes code names are too good not to mention
+  {
+    source: '/heartyeet',
+    destination: '/docs/job-specification/group#stop_after_client_disconnect',
+    permanent: true,
+  },
+
+  // Moved /docs/drivers/external/podman -> /docs/drivers/podman
+  {
+    source: '/docs/drivers/external/podman',
+    destination: '/docs/drivers/podman',
+    permanent: true,
+  },
+
+  // Moved /docs/operating-a-job/ -> /guides/operating-a-job/
+  {
+    source: '/docs/operating-a-job/(index.html)?',
+    destination: 'https://learn.hashicorp.com/collections/nomad/manage-jobs',
+    permanent: true,
+  },
+  {
+    source: '/docs/operating-a-job/accessing-logs.html',
+    destination:
+      'https://learn.hashicorp.com/tutorials/nomad/jobs-accessing-logs',
+    permanent: true,
```

Comment:
This whole block doesnt have the `.html` extension as optional, is that on purpose?